elixir/.tool-versions

```diff
@@ -1,5 +1,7 @@
 # These are used for the dev environment.
 # This should match the versions used in the built product.
-nodejs 22.20.0
-elixir 1.18.4-otp-27
-erlang 27.3.4.1
+nodejs       22.20.0
+elixir       1.18.4-otp-27
+erlang       27.3.4.1
+chromedriver 144.0.7534.0
+chrome       144.0.7534.0 # https://github.com/weizhliu/asdf-chrome.git
```

Comment:
```suggestion
chrome       144.0.7534.0 # https://github.com/firezone/asdf-chrome.git
```

Reviewed the repo and forked it, and I think it would be good to pin it here.

Reply:
How do you pin it? I just added the comment so people know where to get the plugin from.

Reply:
I don't think you can. Just wanted to avoid adding a supply chain dependency that's neither used by many people nor managed by dependabot (to leverage a cooldown period).

Reply:
I think that is a general problem of asdf? Mise has an (optional) lockfile for this reason: https://mise.jdx.dev/dev-tools/mise-lock.html

Reply:
>I think that is a general problem of asdf? Mise has an (optional) lockfile for this reason: https://mise.jdx.dev/dev-tools/mise-lock.html

The problem isn't the version, it's the place where the dependency is coming from. Anyone who installs this plugin grants full home directory access to the owner of that repository and any contributors.

In general I try to "vendor" dependencies that don't seem to be widely used since we very well could be ground zero for the blast radius.

Reply:
I am not against the vendoring per se but a lockfile basically solves same problem: You pin dependencies against the hash you have reviewed and it won't suddenly change without you knowing.

FWIW, `mise` also has things like an HTTP backend. That would allow us to install tools (like Chrome) from official sources by just feeding it a download URL without having to rely on some plugin.